lib/salus.rb

```diff
@@ -64,6 +64,7 @@ def scan(
         raise e if ENV['RUNNING_SALUS_TESTS']
 
         puts "Could not send Salus report: (#{e.class}: #{e.message})"
+        Bugsnag.notify(e)
```

Comment:
What if someone doesn't have bugsnag integrated? Will this fail for them?

Reply:
No, it will not fail.